docs_zh-CN/tools/pytorch2onnx.md

```diff
@@ -0,0 +1,89 @@
+# Pytorch 转 ONNX （试验性的）
+
+<!-- TOC -->
+
+- [Pytorch 转 ONNX （试验性的）](#pytorch-onnx)
+  - [如何将模型从 PyTorch 转换到 ONNX](#id1)
+    - [准备工作](#id2)
+    - [使用方法](#id3)
+  - [支持导出至 ONNX 的模型列表](#onnx)
+  - [提示](#id4)
+  - [常见问题](#id5)
+
+<!-- TOC -->
+
+## 如何将模型从 PyTorch 转换到 ONNX
+
+### 准备工作
+
+1. 请参照 [install.md](https://mmclassification.readthedocs.io/en/latest/install.html#install-mmclassification) 从源码安装 MMClassification。
+2. 安装 onnx 和 onnxruntime。
+
+  ```shell
+  pip install onnx onnxruntime==1.5.1
+  ```
+
+### 使用方法
+
+```bash
+python tools/pytorch2onnx.py \
```

Comment:
The file path has been moved to `mmclassification/tools/deployment/`, we may check this and the English version.